ReSharper.FSharp/src/FSharp.Psi.Services/src/Daemon/Highlightings/FcsErrors.xml

```diff
@@ -669,4 +669,20 @@
     <Behavour overlapResolvePolicy="NONE"/>
   </Warning>
 
+  <Error staticGroup="FSharpErrors" name="UnionCaseDoesNotTakeArguments" ID="FS0725: This union case does not take arguments">
+    <Parameter type="IFSharpPattern" name="pattern"/>
```

Comment:
Could you check if all the tests pass if you use `IParametersOwnerPat` instead of doing a safe cast later in the quick fix? It should simplify the things a bit if it works :slightly_smiling_face:

Reply:
I tried and all the test fails now 😀